### PR TITLE
Homework has been done.

### PR DESCRIPTION
**Pull request by Okkss**

### CODE REVIEW

File: homework/english/lecture1/grammar.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Homework 1, Task 1
+
+Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1.	Your mum is working today?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:5]: It's a question, so "Is your mom working today"

File: homework/english/lecture1/grammar.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+2.	Why are this boys to talking now?
+3.	Where is he to lives?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:7]: not "is". "Does" and without "s" at the end of verb. Where does he live?

File: homework/english/lecture1/grammar.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+4.	Their mother cooks very well?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:8]: question construction

File: homework/english/lecture1/grammar.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+5.	Lili is not rich, she is  not to drives a Mercedes.
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:9]: "doesn't " without "to" and "s"

File: homework/english/lecture1/grammar.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+6.	Let´s go out. It is not raining now.
+7.	I have to go now. It is getting dark
+8.	A young man usually stands at the window.
+9.	Ann usually makes doll dresses herself.
+10.	His wife having coffee at the moment?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:14]: question construction

File: homework/english/lecture1/grammar.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+11.	Let’s go for a walk. It is not raining.
+12.	Look! The boys are coming home.
+13.	Maila likes Chinese food?
+14.	My grandmother is not to work. She is on pension.
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:18]: doesn't work

File: homework/english/lecture1/grammar.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+15.	What is your brother watching now?
+16.	Where are you live?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:20]: do

File: homework/english/lecture1/grammar.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+17.	Are You  play the piano well?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:21]: do. present simple

File: homework/english/lecture1/grammar.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+18.	Where is their brother swimming now?
+19.	What are they to eat at breakfast?
+20.	Are They  swimming at the moment?
+21.	Mike is not wearing a tie today.
+22.	Maila is not to watch TV every morning.
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:26]: does not watch

File: homework/english/lecture1/grammar.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+1.	We are not going to school today.
+2.	What are you doing after school?
+3.	At the moment Peter is working in Russia.
+4.	Does he get a new car?
+5.	He never wears a hat.
+6.	He doesn’t like black coffee.
+7.	We are have a good time.
+8.	What are you doing now?
+9.	It is raining at the moment.
+10.	How are you like the game?
```
Comment (zhdanok) [inline homework/english/lecture1/grammar.md:41]: do

**Review by zhdanok — CHANGES_REQUESTED** [review]
See notes

**Review by zhdanok — APPROVED** [review]
lgfm

**Review by zhdanok — APPROVED** [review]
lgfm

**Review by zhdanok — APPROVED** [review]
lgfm

**Review by zhdanok — APPROVED** [review]

**Review by Okkss — COMMENTED** [review]
lgfm

**Pull request merged**